src/pivot-table/hooks/use-table-rect.ts

```diff
@@ -0,0 +1,9 @@
+import { useMemo } from "react";
+import { LayoutService, Rect } from "../../types/types";
+import { DISCLAIMER_HEIGHT } from "../constants";
+
+export const useTableRect = (rect: Rect, layoutService: LayoutService) =>
+  useMemo(
+    () => (layoutService.hasLimitedData ? { ...rect, height: rect.height - DISCLAIMER_HEIGHT } : rect),
```

Comment:
I think probably the usage of `layoutService.hasLimitedData` + `DISCLAIMER_HEIGHT` subtraction from height, will have side effect on height calculations we do in PivotTable component 

https://github.com/qlik-oss/sn-pivot-table/blob/f2afd5f8f9c1554ebc5064e575188fa665c37959/src/pivot-table/components/PivotTable.tsx#L102-L107